reader.lua

```diff
@@ -52,14 +52,11 @@ else
     end
 end
 
-local Device = require("device")
-local dpi_override = G_reader_settings:readSetting("screen_dpi")
-if dpi_override ~= nil then
-    Device:setScreenDPI(dpi_override)
-end
-
-local CanvasContext = require("document/canvascontext")
-CanvasContext:init(Device)
+-- should check DEBUG option in arg and turn on DEBUG before loading other
+-- modules, otherwise DEBUG in some modules may not be printed.
+local dbg = require("dbg")
```

Comment:
Unless you think it's clearer this way (I don't really think it matters) I'd probably move this part back to where it was to minimize the diff. But whichever way you prefer.

Reply:
I do kinda like it there, that leaves the whole "argument parsing" stuff in a single block without that in between ;).

Reply:
In a way it's part of the argument parsing but I see what you're saying. So yeah, cool. ^_^